src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -288,15 +308,115 @@ private void launchFirefox() {
      */
     private void launchInternetExplorer() {
         setupDrivermanager(InternetExplorerDriverManager.getInstance());
-        DesiredCapabilities capabilities = DesiredCapabilities.internetExplorer();
-        setCapabilities(capabilities);
-        driver = new InternetExplorerDriver(capabilities);
+        String browserName = BrowserType.IE;
+        DesiredCapabilities desiredCapabilities = DesiredCapabilities.internetExplorer();
+        setCapabilitiesForIe(desiredCapabilities, browserName);
+        driver = new InternetExplorerDriver(desiredCapabilities);
         registerShutdownHook(driver);
     }
 
-    private void setCapabilities(DesiredCapabilities capabilities) {
-        capabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
-        capabilities.setCapability("requireWindowFocus", true);
+    private void setCapabilitiesForIe(DesiredCapabilities desiredCapabilities, String browserName) {
```

Comment:
why is this ie specific? could this be generic?

Reply:
The internet explorer has no options :O( , so this is why there is a special treatment.